test/models/ToolpadHome.ts

```diff
@@ -89,4 +89,23 @@ export class ToolpadHome {
 
     return { id: idMatch[1], name };
   }
+
+  async duplicateApplication(appName: string) {
+    await this.page.click(
+      `[role="row"] >> has="input[value='${appName}']" >> [aria-label="Application menu"]`,
```

Comment:
I'd suggest to create a `getAppRow` function that returns a locator for the row, from which you can create a locator for the menu button.

Reply:
we already have this function, could probably use it